### DISCUSSION TITLE
Add gas prices

### DISCUSSION THREAD

**Discussion by TheAdventurer64** [organicmaps/organicmaps#2895] [Ideas]
As I was traveling today, I had an idea: What if we added gas prices?

Adding gas prices is a great idea, as it can help differentiate Organic Maps with other navigation apps that use OSM data. However, it comes with some important questions:

1. Can this feature work in a offline based map?

2. What source can we use that is "organic"? (compatible with OSM license)

3. Do we have enough resources for this to be added?

**Comment by hb0nd** [comment]
Not really practical imo, prices are too volatile & afaik there's no openly licensed petrol-price database
